go/runtime/txpool/txpool.go

```diff
@@ -460,20 +463,31 @@ func (t *txPool) checkTxBatch(ctx context.Context, rr host.RichRuntime) {
 		return
 	}
 
+	// Ensure block round is synced to storage.
+	waitSyncCtx, cancelWaitSyncCtx := context.WithTimeout(ctx, checkTxWaitRoundSyncedTimeout)
+	defer cancelWaitSyncCtx()
+
+	t.logger.Debug("ensuring block round is synced", "round", bi.RuntimeBlock.Header.Round)
+	if _, err = t.history.WaitRoundSynced(waitSyncCtx, bi.RuntimeBlock.Header.Round); err != nil {
+		// Block round isn't synced yet, so make sure the batch check is
+		// retried later to avoid aborting the runtime, as it is not its fault.
+		t.logger.Info("block round is not synced yet, retrying transaction batch check later",
```

Comment:
```suggestion
		t.logger.Warn("block round is not synced yet, retrying transaction batch check later",
```
Should this be `Warn`? :man_shrugging:

Reply:
Probably not, since the user can't really do anything about it?